sharepoint/sharepoint-ps/sharepoint-online/Add-SPOAppPrioritizationPolicy.md

```diff
@@ -0,0 +1,137 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/Add-SPOAppPrioritizationPolicy
+applicable: SharePoint Online
+title: Add-SPOAppPrioritizationPolicy
+schema: 2.0.0
+author: killerewok2000
+ms.author: Sibourda
+ms.reviewer:
+---
+
+# Add-SPOAppPrioritizationPolicy
+
+## SYNOPSIS
+
+Adds a new SPO app prioritization Policy to your tenancy.
+
+## SYNTAX
+
+```powershell
+Add-SPOAppPrioritizationPolicies -AppId <Guid string of the app you need SPAP enabled for> -AzureSubscriptionId <Guid string of the Azure Subscription Id> -ResourceGroup <String> -Account <String> -QuotaMultiplier <int> 
+```
+
+## DESCRIPTION
+
+This cmdlet adds a new SPO app prioritization Billing policies to your tenancy. You must be a SharePoint you must be a SharePoint Tenant Admin to run the commandlet. You need to make sure you’re not adding a new billing policy for an app that already has a billing policy to it. 
```

Comment:
>You must be a SharePoint you must be a SharePoint Tenant Admin to run the commandlet.  [](http://example.com/codeflow?start=80&length=86)

Remove this sentence -- by default only SharePoint Administrators can run SPO PowerShell cmdlets.